plugins/woocommerce/changelog/add-gitignore-custom-cursor-rules

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: dev
+Comment: Exclude custom cursor rules from version control
```

Comment:
We can drop this changelog entry if the top-level configurations do not affect the core end-users.